jest.config.js

```diff
@@ -15,6 +15,12 @@ module.exports = {
   // An array of file extensions your modules use
   moduleFileExtensions: ['js', 'json', 'jsx'],
 
+  // A map from regular expressions to module names that allow to stub out resources 
+  // with a single module
+  moduleNameMapper: {
+    '\\.(css|scss|jpg|png)$': '<rootDir>/ignore-css.js',
```

Comment:
You can use `ignore-module.js` instead since the file handles image files as well